build-packages.cmd

```diff
@@ -34,8 +34,8 @@ if [!processedArgs!]==[] (
 :ArgsDone
 
 call %__ProjectDir%/dotnet.cmd msbuild /nologo /verbosity:minimal /clp:Summary /nodeReuse:false^
-  /p:__BuildOS=Windows_NT /flp:v=detailed;Append;LogFile=build-packages.log^
-  /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log^
+  /p:__BuildOS=Windows_NT /flp:v=detailed;Append`;LogFile=build-packages.log^
+  "'/l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log'"^
```

Comment:
I am not sure what the `BinClashLogger` is but it is coming from the tools directory.  Looks like corefx is using the binary logger. `/bl:Build.binlog` 

The `;` before `Append` needed to be escaped so that powershell didn't treat it as end of command.
For some reason the same didn't work in the BinClashLogger `;LogFile` case.  After experimentation, I resorted to the nested quote escaping.

Reply:
It checks that we don't write different binaries to the same location: https://github.com/dotnet/buildtools/blob/6736870b84e06b75e7df32bb84d442db1b2afa10/src/Microsoft.DotNet.Build.Tasks/BinClashLogger.cs.

It's probably a good idea for us to use binary logs, but they're independent things afaik.